test/WebJobs.Script.Tests/TestScripts/WindowsBatch/WebHookTrigger/run.bat

```diff
@@ -1,3 +1,3 @@
 ﻿echo OFF
-SET /p req=
+SET /p req=<%req%
```

Comment:
I think that will only get the first line of the file. I assume the intent is to get the entire content?